packages/jupyter-ai/jupyter_ai/extension.py

```diff
@@ -406,3 +407,10 @@ def _init_persona_manager(self, ychat: YChat) -> Optional[PersonaManager]:
             self.log.exception(e)
         finally:
             return persona_manager
+
+    def _link_jupyter_server_extension(self, server_app):
+        """Setup custom config needed by this extension."""
+        c = Config()
+        c.ContentsManager.allow_hidden = True
+        server_app.update_config(c)
+        super()._link_jupyter_server_extension(server_app)
```

Comment:
Just curious, what is the motivation for setting this globally? Is it so that users can see `.jupyter` from the JupyterLab UI?

Reply:
Yeah, otherwise there is no way for users to view and edit files in .jupyter.